locales/messages.json

```diff
@@ -663,6 +663,8 @@
   "_ErrorParsingBinaryParagraph.comment": "An example of {spec} is zlib:x64-windows.",
   "ErrorRequireBaseline": "this vcpkg instance requires a manifest with a specified baseline in order to interact with ports. Please add 'builtin-baseline' to the manifest or add a 'vcpkg-configuration.json' that redefines the default registry.",
   "ErrorRequirePackagesList": "`vcpkg install` requires a list of packages to install in classic mode.",
+  "ErrorScriptDownloadFailed": "The script download failed.\n{error_msg}.",
```

Comment:
Newline separators like this should not be in localized constants. What does this prefix add that is not already part of `{error_msg}`?

Reply:
There are many localized messages with newline separators..

Reply:
That said, I can just print the error message directly. It doesn't need to be localized